### ISSUE TITLE
Generating Meta-Panel

### ISSUE THREAD

**Issue by David-Baddeley** [python-microscopy/python-microscopy#20]
**[Original report](https://bitbucket.org/david_baddeley/python-microscopy/issue/20) by Michael Graff (Bitbucket: [Orgual](https://bitbucket.org/Orgual), ).**

----------------------------------------

You can open a first file without problems. If you go to File/Open and select another file (multiple csv files were tested) an error occurs.
It will result in the meta panel not being printed properly but overlapping with the left side panels.

Traceback (most recent call last):
  File "C:\Users\mmg82\PycharmProjects\3drendering\PYME\LMVis\visCore.py", line 315, in OnOpenFile
    self.OpenFile(filename)
  File "C:\Users\mmg82\PycharmProjects\3drendering\PYME\LMVis\visCore.py", line 547, in OpenFile
    self._createNewTabs()
  File "C:\Users\mmg82\PycharmProjects\3drendering\PYME\LMVis\VisGUI.py", line 297, in _createNewTabs
    self.AddPage(self.mdp, caption='Metadata')
  File "C:\Users\mmg82\PycharmProjects\3drendering\PYME\ui\AUIFrame.py", line 56, in AddPage
    self._mgr.Update()
  File "C:\Users\mmg82\AppData\Local\Continuum\Anaconda2\lib\site-packages\wx-3.0-msw\wx\lib\agw\aui\framemanager.py", line 6392, in Update
    self.UpdateNotebook()
  File "C:\Users\mmg82\AppData\Local\Continuum\Anaconda2\lib\site-packages\wx-3.0-msw\wx\lib\agw\aui\framemanager.py", line 6578, in UpdateNotebook
    paneInfo.window.Reparent(notebook)
  File "C:\Users\mmg82\AppData\Local\Continuum\Anaconda2\lib\site-packages\wx-3.0-msw\wx\_core.py", line 16712, in __getattr__
    raise PyDeadObjectError(self.attrStr % self._name)
wx._core.PyDeadObjectError: The C++ part of the MetadataPanel object has been deleted, attribute access no longer allowed.

**Comment by David-Baddeley** [comment]
**Original comment by Michael Graff (Bitbucket: [Orgual](https://bitbucket.org/Orgual), ).**

----------------------------------------

I changed the method that way, that the old meta panel isn't destroyed anymore. That way there's no pointer that refers to a invalid object anymore. The user can close the old tab easily using the middle mouse button.

I changed the issue to minor, since that's not a major bug with this solution. 
We could still try to close this issue completely and closing the tab automatically.
